packages/property-view/src/browser/model-property-widget.tsx

```diff
@@ -24,7 +24,7 @@ export class ModelPropertyWidget extends ReactWidget implements PropertyViewCont
     @inject(ModelService) protected modelService: ModelService;
     @inject(ApplicationShell) protected shell: ApplicationShell;
 
-    protected model: CrossModelRoot | undefined;
+    protected model: CrossModelRoot | undefined = undefined;
```

Comment:
I added it because it's also in the form-editor-widget: https://github.com/CrossBreezeNL/crossmodel/blob/059aee225e24bf1e18f240f7a71163d2124b3a03/packages/form-client/src/browser/form-editor-widget.tsx#L46C1-L47C1

Can it be removed in both places then?

Reply:
In the last commit I removed all unneeded undefined assignments, since undefined is always implicit (based on understanding of https://www.tektutorialshub.com/typescript/null-undefined-in-typescript/).

Reply:
That is correct, thank you!